cfgov/apache/conf.d/alias.conf

```
@@ -2,8 +2,9 @@
 Alias /BingSiteAuth.xml ${STATIC_PATH}/BingSiteAuth.xml
 Alias /code.json ${STATIC_PATH}/code.json
 Alias /data.json ${STATIC_PATH}/data.json
-Alias /favicon.ico ${STATIC_PATH}/favicon.ico
 Alias /financial-intuition.xml ${STATIC_PATH}/financial-intuition.xml
+# Favicon
+Alias /favicon.ico ${STATIC_PATH}/favicon.ico
```

Wait, so is this a favicon?

```suggestion
Alias /favicon.ico ${STATIC_PATH}/favicon.ico
```

---

I'd added the code comment because the way the commenting is sectioned in that file makes it sorta look like all the top things are related to Bing https://github.com/cfpb/consumerfinance.gov/blob/07a78a86fdd22076278153bd0809c7019ea188e4/cfgov/apache/conf.d/alias.conf#L1

I'm  fine with whatever you recommend.